frontend/src/components/HomePage/Middle/PostFeed.js

```diff
@@ -114,11 +121,11 @@ function PostFeed() {
                   ) : (
                     <div />
                   )}
-                  {checkPost && targetedPost.user_id === currentUser.id && (
+                  {checkPost && targetedPost.user_id === sessionUser.id && (
                     <Modal onClose={() => setCheckPost(false)}>
                       <CreatePostModal
                         type="update"
-                        currentUser={currentUser}
+                        currentUser={sessionUser}
```

Comment:
You should probably update this prop name too right?
```suggestion
                        sessionUser={sessionUser}
```

Reply:
just a bad prop name I should fix, old code